pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -257,7 +262,7 @@ def create_cone(self, cs_axis, position, bottom_radius, top_radius, height, name
         cs_axis : str
             Axis of rotation of the starting point around the center point.
             The default is ``None``, in which case the Z axis is used.
-        center_position : list, optional
+        position : list, optional
```

Comment:
@tizianrot we shouldn't change variable names so we should get back to center_position.